docs/manuals/source/TasksAndConcepts/NetworkSetup.rst

```diff
@@ -1,11 +1,53 @@
 Network setup
 =============
 
+.. _section-TCP_KeepAlive:
+
+Heartbeat Interval - TCP Keepalive
+----------------------------------
+
+.. index::
+   single: Broken pipe
+   single: tcp keepalive
+   single: heartbeat interval
+   single: firewall, router
+
+In Bareos the *Heartbeat Interval* is an optional parameter expressed in seconds and
+implemented only on systems that provides the **setsockopt TCP_KEEPIDLE** function
+like Unix, Linux, Windows... The default for those systems is 2 hours (7200s).
+The Bareos default value is zero (0), which means no change is made to the socket.
+If non-zero, a heartbeat signal is send from a daemon to the others to keep the channels
+active.
+
+This feature is particularly useful if you have a router or a firewall that does not
+follow Internet standards and times out an valid connection after a short duration
+despite the fact that :strong:`keepalive` is set.
+This usually results in a *broken pipe* error message.
+
+.. warning::
+
+   A hierarchy exist for this parameter depending of its usage.
+   When :strong:`HeartBeatInverval` is defined at the daemon resource, this will
+   affect all sockets used by this daemon or program. While when used in sub part this
+   will override the global setup and only concern this specific resource.
+   For example :config:option:`dir/director/HeartbeatInterval`\ (global) is overwritten by the
+   more resource specific :config:option:`dir/client/HeartbeatInterval`\ .
+
+.. note::
+
+   Consult `TLDP howto TCP-Keepalive <https://tldp.org/HOWTO/TCP-Keepalive-HOWTO/overview.html>`_
+   to learn how to setup tcp keepalive in you Linux.
+
+
 .. _section-ClientInitiatedConnection:
 
 Client Initiated Connection
 ---------------------------
 
+.. index::
```

Comment:
Do we maybe want to put the Passive Client here, and the Client Initated connection after that? I think that the passive client is probably much more used than the client initiated connection.

Reply:
Make sense, will add that change